universal-login-react/src/ui/Notifications/EmojiPanelWithFakes.tsx

```diff
@@ -8,15 +8,15 @@ interface EmojiPanelWithFakesProps {
 
 export const EmojiPanelWithFakes = ({securityCodeWithFakes, onEmojiClicked}: EmojiPanelWithFakesProps) => {
   const emojis = securityCodeWithFakes.map((code: number, index: number) => (
-    <li key={`securityCodeWithFakes_${index}`} onClick={() => onEmojiClicked(code)} >
+    <li style={{margin: '10px'}} key={`securityCodeWithFakes_${index}`} onClick={() => onEmojiClicked(code)} >
```

Comment:
inline style? bleh!

Reply:
btw that should be addressed in css as sth like:
```css
ul.universal-login-default  li {
  margin: 10px
}
```

btw do we have a wrapping `ul` element for those `li`'s?